README.md

```diff
@@ -1,2 +1,49 @@
-# rfcs
-RFCs for additions or changes to MOXY open-source projects
+# MOXY OSS RFCs
+
+This repository is intended to be used for reviewing submitted RFCs related to additions or changes to MOXY OSS projects and processes.
+
+The MOXY OSS RFC (request for comments) process is intended to provide a consistent way for MOXY to facilitate the transparency of decisions made for features and changes in OSS related projects and processes.
+
+In short, an MOXY RFC is a proposal for a significant change or a new feature.
+
+The RFC process will:
+
+- Facilitate the transparency of decision making by MOXY to end-users, including MOXY members themselves and the community.
+- Enable individual contributors to help make decisions.
+- Allow domain experts to have input in decisions.
+- Manage the risk of decisions made.
+- Have a snapshot of context for the future.
+
```

Comment:
I think we should add two points here:

- Triage future work according to its relevance
- Enable design decisions to be taken before the implementation stage

Reply:
Agree with first point. However the second one is kind of already mentioned, e.g.: `Allow domain experts to have input in decisions.` and `Enable individual contributors to help make decisions.`.